atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionTest.java

```diff
@@ -745,7 +823,38 @@ public void testReadMyWritesColumnRangePagingTransaction() {
     }
 
     @Test
-    public void testReadMyDeletesColumnRangePagingTransaction() {
+    public void testReadMyWritesColumnRangePagingTransaction_iterator() {
+        Transaction t = startTransaction();
```

Comment:
a better/easier/clearer test would be doing the setup as normal, and then assert in the batching visitable view, and then do the iterator view, and then assert that the contents are the same? at the same time, not too fussed

Reply:
yeah, got stuck in this pattern after touching a few tests where it was not appropriate to combine the two

Reply:
I consolidated in the few cases where it was safe to do so without making the test harder to understand or where the it was important to the test itself